src/display.c

```diff
@@ -777,7 +777,7 @@ PyImaging_DrawWmf(PyObject* self, PyObject* args)
     char* ptr;
 
     char* data;
-    int datasize;
+    Py_ssize_t datasize;
```

Comment:
I find this change is unnecessary? I've created https://github.com/nulano/Pillow/pull/11 to demonstrate.

Reply:
See reply at https://github.com/nulano/Pillow/pull/11#issuecomment-720086493